website/docs/components/accordion/partials/specifications/states.md

```diff
@@ -0,0 +1,15 @@
+## States
+
+![Example of accordion states](/assets/components/accordion/accordion-states.png =800x*)
+
+### Contains interactive
```

Comment:
In one cases we use a human-readable text ("Contains interactive") in the other a code-like title (`isOpen`). I think we should stick to one (probably the human-readable type.

Reply:
@cveigt